### ISSUE TITLE
Mismatched dark mode colours (v1.9.1)

### ISSUE THREAD

**Issue by max-ae** [corona-warn-app/cwa-app-ios#1704]
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Technical details

- Device name: iPhone 12 Pro Max
- iOS Version: 14.3
- App Version: 1.9.1

## Describe the bug

See screenshot, dark mode colours in test result screen do not match

## Steps to reproduce the issue
1. Open test result screen with registered test
2. See error

![image](https://user-images.githubusercontent.com/4730112/102385151-cb80e600-3fcd-11eb-8eb3-1b617e00ec1b.png)

----
Internal Tracking ID: EXPOSUREAPP-4226

**Comment by Ein-Tim** [comment]
It is not only about this screen, f.e. also this screen:
![image](https://user-images.githubusercontent.com/67682506/102386045-e1db7180-3fce-11eb-8bbe-024d80353310.png)

**Comment by heinezen** [comment]
Hey @max-ae ,

Thank you for the report. We are aware of the issue and are currently investigating it internally in our Jira ticket system (your ticket ID: EXPOSUREAPP-4226) to find the cause of the UI glitches. Once we get news when an update is ready, we will inform you here on Github.

Regards,
CH

----
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
PR: #1719

Fix will be in 1.11.0 (according to the milestone)

**Comment by Ein-Tim** [comment]
@max-ae 
Can you confirm that this is solved with 1.11?

**Comment by max-ae** [comment]
Yes, I can confirm that this issue is fixed with 1.11.
